get-deps

```diff
@@ -116,6 +116,28 @@ if test -e /etc/arch-release ; then
   exit $?
 fi
 
+# DONOT forget to set USE="X"
```

Comment:
Who is this comment for? Should this script set `USE=X`?

Reply:
For users. It might (or should?) be documented on the website. 

`USE=X` is the system configuration of the user environment.
The script should not set it itself for security reason.
Most graphical Gentoo system users have already enabled this flag in the system. So, this is meddlesomeness.

Reply:
I think "no one" will see that comment, so it should probably be removed from the script and either:

* Add something to the script that checks for it being set and generates a warning if it isn't
* Cover that this should be set in the documentation

Reply:
I have added error message if X is not found in USE flags